content-api/content-actors/src/main/scala/org/sunbird/content/actors/EventActor.scala

```diff
@@ -174,4 +175,33 @@ class EventActor @Inject()(implicit oec: OntologyEngineContext, ss: StorageServi
       })
     }).flatMap(f => f)
   }
+
+  override def systemUpdate(request: Request): Future[Response] = {
+    RedisCache.delete(request.get("identifier").asInstanceOf[String])
+    val identifier = request.get("identifier").asInstanceOf[String]
+    val updatedIdentifier = if (!identifier.endsWith(".img")) s"$identifier.img" else identifier
+    DataNode.read(request).flatMap { node =>
+      // Extract attributes to be updated from the request body
+      val attributesToUpdate = request.getRequest.asInstanceOf[java.util.Map[String, AnyRef]]
+
+      // Append attributes from the request to the node's metadata
+      attributesToUpdate.forEach(new java.util.function.BiConsumer[String, AnyRef] {
+        override def accept(key: String, value: AnyRef): Unit = {
+          node.getMetadata.put(key, value)
+        }
+      })
+      // Save the updated node
+      DataNode.updatev2(request, _ => node).recover {
```

Comment:
this method implementation does replaces .img if it exists in the identifier -- which should not be the case for system/update APIs... 
by mistake if this method called using object do_id.img -- then it would edit the details in "do_id" which is Live ??